src/policy/prelude.md

```diff
@@ -0,0 +1,56 @@
+Policy for inclusion in the prelude
+===================================
+
+The Rust standard library provides a "prelude": items that Rust programs can
+use without having to import anything. For instance, the Rust prelude includes
+[`Vec`](https://doc.rust-lang.org/std/vec/struct.Vec.html), so programs can
+write `Vec::with_capacity(4)` without having to first import `Vec` from the
+standard library.
+
+Each edition of Rust has its own prelude, so that new editions of Rust may add
+items without making those items available to all code in older editions of
+Rust.
+
+This policy sets out the requirements for what items we add to the prelude, and
+whether we add those items to the prelude for a future edition or to the common
+prelude for all Rust editions.
+
+When to use editions
+--------------------
+
+Adding a trait to the prelude can break existing Rust code, by creating name
+resolution ambiguity where none previously existed. While introducing
+resolution ambiguities may be "permitted breakage", doing so is quite
+disruptive, and we want to avoid doing so. Thus, we generally never add a trait
+to the common prelude; we only add traits to the prelude for a new edition of
+Rust.
+
+Adding items *other* than traits to the prelude will never produce conflicts or
+other compatibility issues with existing code, since we allow shadowing and
```

Comment:
```suggestion
Adding items *other* than traits to the prelude introduces less, but not zero,
name resolution conflicts with existing code, since we allow shadowing and
```

Reply:
I've fixed this with a substantial expansion to the documentation.